<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1642</width>
    <height>834</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2e3349;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="7,1,2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
            <item>
             <widget class="QTextEdit" name="explain_text_edit">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Baskerville Old Face&quot;;</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="PlotWidget" name="graph" native="true">
              <property name="styleSheet">
               <string notr="true">background: #707DB3;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="leftMargin">
             <number>150</number>
            </property>
            <property name="rightMargin">
             <number>100</number>
            </property>
            <item>
             <widget class="QCheckBox" name="show_type_i">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Baskerville Old Face</family>
                <pointsize>15</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #58ccb4;font: 15pt &quot;Baskerville Old Face&quot;;</string>
              </property>
              <property name="text">
               <string>Show Type I Error</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="show_type_ii">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Baskerville Old Face</family>
                <pointsize>15</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #58ccb4;font: 15pt &quot;Baskerville Old Face&quot;;</string>
              </property>
              <property name="text">
               <string>Show Type II Error</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="show_power">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Baskerville Old Face</family>
                <pointsize>15</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #58ccb4;font: 15pt &quot;Baskerville Old Face&quot;;</string>
              </property>
              <property name="text">
               <string>Show Power</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="leftMargin">
               <number>100</number>
              </property>
              <property name="topMargin">
               <number>15</number>
              </property>
              <property name="rightMargin">
               <number>200</number>
              </property>
              <property name="bottomMargin">
               <number>15</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <family>Baskerville Old Face</family>
                    <pointsize>15</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: #FFCA6C;
font: 15pt &quot;Baskerville Old Face&quot;;
</string>
                  </property>
                  <property name="text">
                   <string>Null Hypothesis :</string>
                  </property>
                  <property name="textFormat">
                   <enum>Qt::RichText</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="h0mu">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: #4a4f63;
border-color: rgb(4, 4, 4);
font: 12pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="font">
                   <font>
                    <family>Baskerville Old Face</family>
                    <pointsize>15</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: #FFCA6C;
font: 15pt &quot;Baskerville Old Face&quot;;

</string>
                  </property>
                  <property name="text">
                   <string>True Parameter :</string>
                  </property>
                  <property name="textFormat">
                   <enum>Qt::RichText</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="hamu">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: #4a4f63;
border-color: rgb(4, 4, 4);
font: 12pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="font">
                   <font>
                    <family>Baskerville Old Face</family>
                    <pointsize>15</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: #FFCA6C;
font: 15pt &quot;Baskerville Old Face&quot;;
</string>
                  </property>
                  <property name="text">
                   <string>Type I Error :     </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="type_i_error">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: #4a4f63;
border-color: rgb(4, 4, 4);
font: 12pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="leftMargin">
               <number>15</number>
              </property>
              <property name="topMargin">
               <number>30</number>
              </property>
              <property name="rightMargin">
               <number>200</number>
              </property>
              <property name="bottomMargin">
               <number>30</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="styleSheet">
                   <string notr="true">color: #FFCA6C;
font: 15pt &quot;Baskerville Old Face&quot;;</string>
                  </property>
                  <property name="text">
                   <string>Type II Error :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="type_ii_error">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: #4a4f63;
border-color: rgb(4, 4, 4);
font: 12pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="styleSheet">
                   <string notr="true">color: #FFCA6C;
font: 15pt &quot;Baskerville Old Face&quot;;</string>
                  </property>
                  <property name="text">
                   <string>Power :            </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="power">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: #4a4f63;
border-color: rgb(4, 4, 4);
font: 12pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1642</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>h0mu</tabstop>
  <tabstop>hamu</tabstop>
  <tabstop>type_i_error</tabstop>
  <tabstop>show_type_i</tabstop>
  <tabstop>show_type_ii</tabstop>
  <tabstop>show_power</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
